pkg/agent/openflow/pipeline.go

```diff
@@ -454,388 +504,472 @@ func (c *client) DeleteOFEntries(ofEntries []binding.OFEntry) error {
+		ClassifierTable.ofTable.BuildFlow(priorityHigh).
+			Cookie(cookieID).
+			MatchInPort(config.UplinkOFPort).
+			MatchDstMAC(dstMAC).
+			Action().LoadRegMark(FromUplinkRegMark).
+			Action().GotoStage(stageConntrackState).
+			Done(),
+		// This generates the flow to mark the packets from bridge local port.
+		ClassifierTable.ofTable.BuildFlow(priorityHigh).
+			Cookie(cookieID).
+			MatchInPort(config.BridgeOFPort).
+			MatchDstMAC(dstMAC).
+			Action().LoadRegMark(FromBridgeRegMark).
+			Action().GotoStage(stageConntrackState).
+			Done(),
 	}
-	return table.BuildFlow(priorityHigh).
-		MatchRegFieldWithValue(CustomReasonField, CustomReasonDNS).
-		Cookie(c.cookieAllocator.Request(cookie.Default).Raw()).
-		Action().ResubmitToTable(L2ForwardingCalcTable.GetID()).
-		Done()
 }
 
-// dnsResponseBypassPacketInFlow generates a flow which is used to bypass the
-// dns packetIn conjunction flow for dns response packetOut. This packetOut
-// should be sent directly to the requesting client without being intercepted
-// again.
-func (c *client) dnsResponseBypassPacketInFlow() binding.Flow {
-	// TODO: use a unified register bit to mark packetOuts. The pipeline does not need to be
-	// aware of why the packetOut is being set by the controller, it just needs to be aware that
-	// this is a packetOut message and that some pipeline stages (conntrack, policy enforcement)
-	// should therefore be skipped.
-	return AntreaPolicyIngressRuleTable.BuildFlow(priorityDNSBypass).
-		MatchRegFieldWithValue(CustomReasonField, CustomReasonDNS).
-		Cookie(c.cookieAllocator.Request(cookie.Default).Raw()).
-		Action().ResubmitToTable(L2ForwardingOutTable.GetID()).
-		Done()
-}
-
-func (c *client) conntrackBasicFlows(category cookie.Category) []binding.Flow {
+// Feature: featurePodConnectivity
+// Stage: stageConntrackState
+// Tables: ConntrackTable, ConntrackStateTable
+// Stage: stageConntrack
+// Tables: ConntrackCommitTable
+// Refactored from:
+//   - func (c *client) connectionTrackFlows(category cookie.Category) []binding.Flow
+//   - func (c *client) conntrackBasicFlows(category cookie.Category) []binding.Flow
+//   - part of func (c *client) serviceLBBypassFlows(ipProtocol binding.Protocol) []binding.Flow
+// Modifications:
+//   - Remove the flows related with Service since they are for feature Service.
+// conntrackFlows generates the flows about conntrack for feature PodConnectivity.
+func (f *featurePodConnectivity) conntrackFlows() []binding.Flow {
+	cookieID := f.cookieAllocator.Request(f.category).Raw()
 	var flows []binding.Flow
-	for _, proto := range c.ipProtocols {
-		ctZone := CtZone
-		if proto == binding.ProtocolIPv6 {
-			ctZone = CtZoneV6
-		}
+	for _, ipProtocol := range f.ipProtocols {
 		flows = append(flows,
-			ConntrackStateTable.BuildFlow(priorityLow).MatchProtocol(proto).
-				MatchCTStateInv(true).MatchCTStateTrk(true).
+			// This generates the flow to transform the packets from tracked connection in CT zone.
```

Comment:
what does "transform packets from tracked connection in CT zone"?

Reply:
Updated as the following:
```
// This generates the flow to transform the destination IP of request packets or source IP of reply packets
// from tracked connections in CT zone.
```